plugins/inputs/kafka_consumer/kafka_consumer_test.go

```diff
@@ -484,58 +484,24 @@ func TestKafkaRoundTripIntegration(t *testing.T) {
 	}{
 		{"connection strategy startup", "startup", []string{"Test"}, nil, config.Duration(0)},
 		{"connection strategy defer", "defer", []string{"Test"}, nil, config.Duration(0)},
-		{"topic regexp", "startup", nil, []string{"T*"}, config.Duration(5 * time.Second)},
 	}
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			t.Logf("rt: starting network")
 			ctx := context.Background()
-			networkName := "telegraf-test-kafka-consumer-network"
-			network, err := testcontainers.GenericNetwork(ctx, testcontainers.GenericNetworkRequest{
-				NetworkRequest: testcontainers.NetworkRequest{
-					Name:           networkName,
-					Attachable:     true,
-					CheckDuplicate: true,
-				},
-			})
+			kafkaContainer, err := kafkacontainer.RunContainer(ctx,
+				kafkacontainer.WithClusterID("test-cluster"),
+				testcontainers.WithImage("confluentinc/confluent-local:7.5.0"),
+			)
 			require.NoError(t, err)
 			defer func() {
-				require.NoError(t, network.Remove(ctx), "terminating network failed")
+				if err := kafkaContainer.Terminate(ctx); err != nil {
+					require.NoError(t, err)
+				}
 			}()
```

Comment:
Not sure it is a good idea to check the return value in the deferred function. How about
```golang
			defer kafkaContainer.Terminate(ctx)
```
instead?

Reply:
done